docs/source/config_file.rst

```diff
@@ -237,7 +237,7 @@ section of the command line docs.
 
           [tool.mypy]
           exclude = [
-              "^one\.py$",  # TOML's double-quoted strings require escaping backslashes
+              "^one\\.py$",  # TOML's double-quoted strings require escaping backslashes
```

Comment:
Should the next line change too? Haven't tested this myself

Reply:
Yeah, single quotes behave differently, so the current example seems correct.